Cargo.toml

```diff
@@ -9,8 +9,8 @@ members = [
 [workspace.package]
 # Edition 2021 is chosen to support async.
 edition = "2021"
-# Version chosen only because 1.78.0 is the stable version when writing this Cargo.toml
-# If this version is changed, amend the version set in `build.yml`
+# Version 1.80.1 or higher is not compatible because it enforces check of `cfg` names and values. The dependency `blockifier` currently uses unknown values for `target_pointer_width`.
```

Comment:
This has been filed as an issue with blockifier, right?

Reply:
They have sorted it out with `sequencer`